lib/matplotlib/tests/test_font_manager.py

```diff
@@ -24,6 +24,7 @@ def test_font_priority():
 
 def test_json_serialization():
     with tempfile.NamedTemporaryFile() as temp:
+        temp.close()
```

Comment:
From docs:
> If _delete_ is true (the default), the file is deleted as soon as it is closed.

That sounds like the temporary file would be cleaned up here. Then the write in the next line will be to a file that _isn't_ cleaned up.

Since `json_load`/`json_dump` are new, maybe they should be modified to take a file object instead?